### DISCUSSION TITLE
Native fetch in CSR only app

### DISCUSSION THREAD

**Discussion by pboguslawski** [sveltejs/kit#8613] [Q&A]
Warning
```
Loading [...] using `window.fetch`. For best results, use the `fetch` that is passed to your `load` function: https://kit.svelte.dev/docs/load#making-fetch-requests
```
is logged on console when using `window.fetch` in app prepared for CSR only (SPA compiled using static adapter).

Didn't find any differences between provided fetch and window.fetch on https://kit.svelte.dev/docs/load#making-fetch-requests for CSR.

Any reason to use provided fetch instead of window.fetch in CSR only scenario?

**Comment by teemingc** [comment]
The only benefit seems to be an easy transition if you ever decide to utilise SSR.
